contentcuration/contentcuration/migrations/0069_auto_20170605_2122.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-06-06 04:22
+from __future__ import unicode_literals
+
+import contentcuration.models
+from django.db import migrations
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contentcuration', '0068_auto_20170519_1337'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='contentnode',
+            name='content_id',
+            field=contentcuration.models.UUIDField(db_index=True, default=uuid.uuid4, editable=False, max_length=32),
```

Comment:
yeah NVM it looks like there's no indication that indices *have* to be unique, just small compared to the # of the table entries.